cmd/soci/commands/push.go

```diff
@@ -92,6 +92,9 @@ if they are available in the snapshotter's local content store.
 		}
 
 		indexDesc := indexDescriptors[len(indexDescriptors)-1]
+		if !indexDesc.IsORASManifest {
```

Comment:
What's missing that prevents pushing ORT manifests? Lack of library support?

Reply:
Yes.